packages/sveltekit/src/generators/application/files/eslintrc.cjs__template__

```diff
@@ -7,7 +7,7 @@ module.exports = {
     'prettier',
   ],
   plugins: ['svelte3', '@typescript-eslint'],
-  ignorePatterns: ['!**/*', '*.cjs'],
+  ignorePatterns: ['!**/*', '*.cjs', 'vite.config.ts'],
```

Comment:
I notice today we need to also add local node_modules here too. Its part of the next version im doing for the others. But saying you moving this now best to add it from start

Reply:
Okay, thanks